acq4/devices/LightSource/LightSource.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+from acq4.devices.Device import *
+from PyQt4 import QtCore, QtGui
+import acq4.util.Mutex as Mutex
+
+class LightSource(Device):
+    """Simple device which reports information of current illumination source."""
+
+    sigLightChanged = QtCore.Signal(object) # to be used upstream 
+    
+    def __init__(self, dm, config, name):
+        Device.__init__(self, dm, config, name)
+        # self.lightsourceconfig = config.get('sources')
+        self.sourceState = {}
+        self.lock = Mutex.Mutex()
+
+    def describe(self):
```

Comment:
In my original suggestion (https://github.com/aiephys/acq4/issues/3#issuecomment-257026585), the `describe()` method would only return information about lights that were currently active. It doesn't _have_ to be that way, but the info stored with images probably doesn't need to describe lamps that are inactive.